controllers/tutorialsControllers.js

```diff
@@ -21,27 +22,33 @@ const createTutorial = async (req, res) => {
     published: req.body.published,
   });
   try {
+    await postTutorialSchema.validateAsync(req.body);
     await tutorial.save();
     res.status(201).send();
   } catch (error) {
-    res.status(500).send({ message: "Internal Server Error" });
+    if (error) {
+      res.status(400).send({ message: "Invalid Tutorial Details" });
+    } else {
+      res.status(500).send({ message: "Internal Server Error" });
+    }
   }
 };
 
 // Update a Tutorial by ID
 const updateTutorial = async (req, res) => {
   try {
     const { title, description, published } = req.body;
+    const { error, body: tutorialBody } = postTutorialSchema.validate(req.body);
```

Comment:
We should return the error here, no need to always return error from catch block. 
```
if (error) {
   return res.status(400).send({ message: <error-detail> });
}
```

Above code will return 400 error if there will be any validation errors.

CC: @JainishaParmar @ashishkava